### DISCUSSION TITLE
What should I do if the Proton-Drive app works if installed outside of brew?

### DISCUSSION THREAD

**Discussion by zhao-li** [Homebrew/discussions#5355] [Casks]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.2.21
ORIGIN: https://github.com/Homebrew/brew
HEAD: 82c2e743a5bcea725f9ca1429e3e21c3088ff904
Last commit: 5 days ago
Core tap JSON: 11 May 09:53 UTC
Core cask tap JSON: 11 May 09:53 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 3.1.4 => /opt/homebrew/Library/Homebrew/vendor/portable-ruby/3.1.4/bin/ruby
CPU: octa-core 64-bit arm_firestorm_icestorm
Clang: 15.0.0 build 1500
Git: 2.45.0 => /opt/homebrew/bin/git
Curl: 8.4.0 => /usr/bin/curl
macOS: 14.4.1-arm64
CLT: 15.3.0.0.1.1708646388
Xcode: N/A
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: Some installed casks are deprecated or disabled.
You should find replacements for the following casks:
  authy
```

### Description of issue

Hi 👋 

Thank you for creating this awesome product.

I am not sure if this is the right place to ask.  Please let me know if there's a better place to post this question.

If I download the DMG installation file for Proton Drive from this page: https://proton.me/drive/download
The application sets up and runs properly.

But when I install via `brew install proton-drive`, the application cannot create the Proton Drive folder in the file system.

Both versions were 1.4.0.8600.

Any ideas on what I should do to further troubleshoot?

Thank you for your time and help 🙏 

### Relevant casks

proton-drive

**Comment by bevanjkay** [comment]
Can you please explain what you mean by, "cannot create the Proton Drive folder in the file system"?

**Reply by zhao-li** [reply]
Thank you for your help on this.

When I install by manually downloading the DMG file, the installation process automatically creates the "Proton Drive" folder depicted in the following image from this page: https://proton.me/support/de/proton-drive-macos-setup#:~:text=You%20can%20download%20the%20app,will%20guide%20you%20with%20instructions.&text=Drag%20and%20drop%20the%20Proton%20Drive%20icon%20into%20the%20Applications%20folder.

![image](https://github.com/Homebrew/discussions/assets/2177879/6788bb2a-4af9-4c64-8ce4-42e347224ba4)

When I install it via brew, that folder is not automatically created.

Thank you again for your help on this 🙏
